### ISSUE TITLE
Interaction keybindings broken by relocation of InteractUnit API

### ISSUE THREAD

**Issue by Meorawr** [Stanzilla/WoWUIBugs#372]
The `INTERACTMOUSEOVER` and `INTERACTTARGET` keybindings are broken in the 3.4.1 PTR client as the InteractUnit function has been renamed to [C_PlayerInteractionManager.InteractUnit](https://wowpedia.fandom.com/wiki/API_C_PlayerInteractionManager.InteractUnit) without the existing references to the old function in all of the FrameXML Bindings.xml files being updated to accommodate the change.

# Test Case

1. Open the Keybindings interface and bind either the "Interact With Mouseover" or "Interact With Target" actions in the "Targeting" category.
2. Target or mouseover something and smash the keybinding.
3. Receive script error:

```
Message: [string "INTERACTMOUSEOVER"]:1: attempt to call global 'InteractUnit' (a nil value)
Time: Sun Dec 18 15:04:42 2022
Count: 2
Stack: [string "INTERACTMOUSEOVER"]:1: attempt to call global 'InteractUnit' (a nil value)
[string "INTERACTMOUSEOVER"]:1: in function <[string "INTERACTMOUSEOVER"]:1>

Locals: keystate = "down"
(*temporary) = nil
(*temporary) = "mouseover"
(*temporary) = "attempt to call global 'InteractUnit' (a nil value)"
```
```
Message: [string "INTERACTTARGET"]:1: attempt to call global 'InteractUnit' (a nil value)
Time: Sun Dec 18 15:05:06 2022
Count: 1
Stack: [string "INTERACTTARGET"]:1: attempt to call global 'InteractUnit' (a nil value)
[string "INTERACTTARGET"]:1: in function <[string "INTERACTTARGET"]:1>

Locals: keystate = "down"
(*temporary) = nil
(*temporary) = "anyinteract"
(*temporary) = "attempt to call global 'InteractUnit' (a nil value)"
```

**Comment by Duugu** [comment]
Still not fixed with 3.4.1.4746. :/
If that bug keeps unfixed it will more or less completely break the gameplay for blind players.

**Comment by Meorawr** [comment]
Fixed in 3.4.1.47612.
